src/screens/CitySelectionScreen.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Alert } from 'react-native';
+
+import {
+  Button,
+  DropdownSelect,
+  EmptyMessage,
+  HtmlView,
+  LoadingSpinner,
+  SafeAreaViewFlex,
+  ServiceTiles,
+  Title,
+  Touchable,
+  Wrapper,
+  WrapperRow
+} from '../components';
+import { colors, Icon, normalize } from '../config';
+import { addToStore, readFromStore, removeFromStore } from '../helpers';
+import { useStaticContent } from '../hooks';
+import { DropdownProps } from '../types';
+
+const SELECTED_CITY = 'selectedCity';
+
+const useCitySelection = () => {
+  const [storedCity, setStoredCity] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+
+  const loadStoredCity = useCallback(async () => {
+    setLoading(true);
+    const city = await readFromStore(SELECTED_CITY);
+    setStoredCity(city);
+    setLoading(false);
+
+    return city;
+  }, []);
+
+  const storeCity = useCallback(async (city: string | null) => {
+    if (city) {
+      await addToStore(SELECTED_CITY, city);
+      setStoredCity(city);
+    }
+  }, []);
+
+  const resetCity = useCallback(async () => {
+    await removeFromStore(SELECTED_CITY);
+    setStoredCity(null);
+  }, []);
```

Comment:
it is the first time that the store is used per hook instead of helper, right? what is the benefit of this?

Reply:
I fixed this mess with the last commit. Now we have helper functions instead of hooks.